### PR TITLE
Simplify Windows build instructions

### PR DESCRIPTION
**Pull request by winstliu**
Take two!

### CODE REVIEW

File: docs/build-instructions/windows.md [hunk 1]
```diff
@@ -36,28 +36,18 @@ To also install the newly built application, use `script\build --create-windows-
...
+  * If you just installed Node.js, you'll need to restart Command Prompt before the `node` command is available on your Path.
 
 * `msbuild.exe failed with exit code: 1`
-   * Ensure you have Visual C++ support installed. Go into Add/Remove Programs, select Visual Studio and press Modify and then check the Visual C++ box.
+   * If you installed Visual Studio, ensure you have Visual C++ support installed. Go into Add/Remove Programs, select Visual Studio, press Modify, and then check the Visual C++ box.
+   * If you installed Visual C++ Build Tools, ensure you have Windows 8 SDK support installed.  Go into Add/Remove Programs, select Visual Studio, press Modify and then check the Windows 8 SDK box.
 
 * `script\build` stop with no error or warning shortly after displaying the versions of node, npm and Python
-  * Make sure that the path where you have checked out Atom does not include a space. e.g. use `c:\atom` and not `c:\my stuff\atom`
+  * Make sure that the path where you have checked out Atom does not include a space (use `C:\atom` and not `C:\my stuff\atom`).
```
Comment (maxbrunsfeld) [inline docs/build-instructions/windows.md]: I can see why you removed ".e.g" here; it's a bit weird to use that at the beginning of a sentence. But I think it's nice to make it crystal clear that this is just an example. I would probably say "For example, use `C:\atom`, not `C:my stuff\atom`."

**Comment by maxbrunsfeld** [comment]
:zap: This is much more streamlined. Just one suggestion ☝️ .

File: docs/build-instructions/windows.md [hunk 2]
```diff
@@ -44,10 +44,10 @@ To also install the newly built application, use `script\build --create-windows-
 
 * `msbuild.exe failed with exit code: 1`
    * If you installed Visual Studio, ensure you have Visual C++ support installed. Go into Add/Remove Programs, select Visual Studio, press Modify, and then check the Visual C++ box.
-   * If you installed Visual C++ Build Tools, ensure you have Windows 8 SDK support installed.  Go into Add/Remove Programs, select Visual Studio, press Modify and then check the Windows 8 SDK box.
+   * If you installed Visual C++ Build Tools, ensure you have Windows 8 SDK support installed. Go into Add/Remove Programs, select Visual Studio, press Modify and then check the Windows 8 SDK box.
 
 * `script\build` stop with no error or warning shortly after displaying the versions of node, npm and Python
-  * Make sure that the path where you have checked out Atom does not include a space (use `C:\atom` and not `C:\my stuff\atom`).
+  * Make sure that the path where you have checked out Atom does not include a space. For example, use `C:\atom` instead of `C:\my stuff\atom`).
```
Comment (maxbrunsfeld) [inline docs/build-instructions/windows.md]: :fire: ')'

Comment (50Wliu) [inline docs/build-instructions/windows.md]: 🙀

**Pull request merged**